src/Utilities/Blaze.hpp

```diff
@@ -41,4 +38,75 @@ constexpr bool useDefaultInitialization = false;
 
 // Disable HPX parallelization
 #define BLAZE_HPX_PARALLEL_MODE 0
+
+// Disable all padding
+#define BLAZE_USE_PADDING 0
+
+#define BLAZE_USE_STREAMING 1
+#define BLAZE_USE_OPTIMIZED_KERNELS 1
+#define BLAZE_USE_DEFAULT_INITIALIZATON 0
 /// \endcond
+
+namespace blaze {
+template <typename T>
+BLAZE_ALWAYS_INLINE SIMDdouble step_function(const SIMDf64<T>& v) noexcept
```

Comment:
Could you add a few comments why these additions are necessary?

Reply:
It's less of an addition, and more just copying over the last piece of PointerVector.hpp that we can't immediately discard, but I can add a code comment explaining why we need it.